conda_build/cli/main_build.py

```diff
@@ -601,3 +597,7 @@ def main():
         )
         sys.exit(1)
     return
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
Do we need this?

Reply:
We don't _need_ this, I added it solely to be consistent (only some of the subcommands were previously callable via `python -m`)

but since it's a deprecated interface it probably doesn't make sense to enable the ability now...I'll remove these